packages/astro/src/core/build/types.ts

```diff
@@ -22,6 +22,7 @@ export interface PageBuildData {
 	moduleSpecifier: string;
 	css: Map<string, { depth: number; order: number }>;
 	contentCollectionCss: Map<string, Set<string>>;
```

Comment:
nit: Meant to rename `contentCollectionCss` to `propagatedCss` or `propagatedStyles` for consistency. Possible to add to this PR?